ktor-server/ktor-server-core/jvmAndNix/src/io/ktor/server/request/ApplicationReceiveFunctions.kt

```diff
@@ -58,7 +58,8 @@ public suspend inline fun <reified T : Any> ApplicationCall.receiveOrNull(): T?
  * @return instance of [T] received from this call.
  * @throws ContentTransformationException when content cannot be transformed to the requested type.
  */
-public suspend inline fun <reified T : Any> ApplicationCall.receive(): T = receiveNullable(typeInfo<T>())!!
+public suspend inline fun <reified T : Any> ApplicationCall.receive(): T = receiveNullable(typeInfo<T>())
+    ?: throw CannotTransformContentToTypeException(typeInfo<T>().kotlinType!!)
```

Comment:
maybe extract `typeInfo<T>()` to val?

Reply:
I would keep it as one-liner cause `typeInfo` is compile time constant